telemetry/definitions/commonDefinitions.json

```
@@ -773,6 +773,36 @@
             "name": "codecatalyst_devEnvironmentWorkflowStep",
             "type": "string",
             "description": "Workflow step name"
+        },
+        {
+            "name": "authConnectionsCount",
+            "type": "int",
+            "description": "Current number of authentication connections the user has"
+        },
+        {
+            "name": "authUiElement",
+            "type": "string",
+            "description": "Feature tab user is interacting in.",
+            "allowedValues": [
+                "awsResource",
+                "codewhisperer",
+                "codecatalyst"
+            ]
+        },
+        {
+            "name": "unlockedTabs",
```

this should probably be renamed. 

- if the New Connection page is implemented in other IDEs, the ui element might not be "tabs".
    - (these new definitions are in `commonDefinitions.json` which implies they are relevant to multiple IDEs)

```suggestion
            "name": "authActivated",
```